docs/docs/configuration/generic-mapping.md

```diff
@@ -7,7 +7,10 @@ description: Create a generic mapping method
 
 Mapperly supports generic user defined mapping methods.
 Mapperly implements this mapping method
-using all mappings the user defined in the mapper.
+using all mappings the user defined in the mapper that:
+
+- satisfy generic constraints,
+- and can be substituted instead of generic mapper
```

Comment:
I don't really understand the second part `and can be substituted instead of generic mapper` (probably due to my bad english 🤔 🙈 )... Couldn't this be generalized by writing `... using all mappings the user defined in the mapper that satisfy the source and target type constraints.`

Reply:
I had an association with a math term substitution. Replace one with another without making it invalid. But I am not a native speaker either, so it can be a rubbish idea 😄